server/utils/utils.ts

```diff
@@ -21,3 +23,12 @@ export const initialiseName = (fullName?: string): string | null => {
   const array = fullName.split(' ')
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
+
+export const formatDate = (dateToFormat: string, dateFormat = 'd MMMM yyyy'): string | null => {
```

Comment:
Not strictly part of this PR, but I've moved the implementation of this function from where the filter definitions are in `nunjucksSetup.ts` to here (and then called it in the filter's definition). This means the same function can be used in app code as `formatDate(...)` and in templates via `"date string" | formatDate`